src/bp/migrations/v12_0_0-1560882898-move_users_to_db.ts

```diff
@@ -0,0 +1,70 @@
+import * as sdk from 'botpress/sdk'
+import { ConfigProvider } from 'core/config/config-loader'
+import Database from 'core/database'
+import { StrategyUserTable } from 'core/database/tables/server-wide/strategy_users'
+import { StrategyUsersRepository } from 'core/repositories/strategy_users'
+import { WorkspaceUsersRepository } from 'core/repositories/workspace_users'
+import { GhostService } from 'core/services'
+import { Migration } from 'core/services/migration'
+import { TYPES } from 'core/types'
+import { Container } from 'inversify'
+import _ from 'lodash'
+
+const migration: Migration = {
+  info: {
+    description: 'Copy existing users from workspaces file to database',
+    type: 'database'
+  },
+  up: async (
+    bp: typeof sdk,
+    configProvider: ConfigProvider,
+    database: Database,
+    inversify: Container
+  ): Promise<sdk.MigrationResult> => {
+    const ghost = inversify.get<GhostService>(TYPES.GhostService)
+    const userRepo = inversify.get<StrategyUsersRepository>(TYPES.StrategyUsersRepository)
+    const workspaceRepo = inversify.get<WorkspaceUsersRepository>(TYPES.WorkspaceUsersRepository)
+
+    const strategyTable = new StrategyUserTable()
+    await strategyTable.createStrategyTable(database.knex, `strategy_default`)
+
+    try {
+      const workspaces: any = await ghost.global().readFileAsObject('/', `workspaces.json`)
```

Comment:
I don't remember, did we decide to keep the super admin logic in the botpress.config file ?